README.md

```diff
@@ -22,12 +22,13 @@ data by [Hypothesis](https://hypothesis.works/) and
 To create traceback database with monkeytype:
 
 ```
-pipenv run monkeytype run <myscript.py>
+pipenv run monkeytype run example/termfrequency/tf_objectoriented.py
```

Comment:
I think user should cd into the example first so that the database wouldn't be created in the main directory. Then run `pipenv run monkeytype run termfrequency/runtermfreq.py inputs/input.txt`